packages/pocket-ic/BUILD.bazel

```diff
@@ -66,53 +67,54 @@ rust_test(
     deps = [":pocket-ic"] + DEPENDENCIES + TEST_DEPENDENCIES,
 )
 
-rust_test_suite(
+test_using_pocket_ic_server(
     name = "test",
     size = "small",
     srcs = ["tests/tests.rs"],
     data = [
         "tests/counter.wasm",
         "tests/icp_ledger.wasm",
         ":test_canister.wasm",
-        "//rs/pocket_ic_server:pocket-ic-server",
     ],
     env = {
-        "POCKET_IC_BIN": "$(rootpath //rs/pocket_ic_server:pocket-ic-server)",
         "COUNTER_WASM": "packages/pocket-ic/tests/counter.wasm",
         "LEDGER_WASM": "packages/pocket-ic/tests/icp_ledger.wasm",
         "TEST_WASM": "$(rootpath :test_canister.wasm)",
     },
+    # Override the default ["manual"] extra_HEAD_tags to ensure this test runs on CI.
+    extra_HEAD_tags = [],
+    # Override the default [] extra_mainnet_tags to skip this test on CI.
+    extra_mainnet_tags = ["manual"],
     flaky = False,
+    macro = rust_test_suite,
     proc_macro_deps = MACRO_DEPENDENCIES,
     deps = [":pocket-ic"] + DEPENDENCIES + TEST_DEPENDENCIES,
 )
 
-rust_test_suite(
+test_using_pocket_ic_server(
     name = "restart",
     size = "medium",
     srcs = ["tests/restart.rs"],
-    data = [
-        "//rs/pocket_ic_server:pocket-ic-server",
-    ],
-    env = {
-        "POCKET_IC_BIN": "$(rootpath //rs/pocket_ic_server:pocket-ic-server)",
-    },
+    # Override the default ["manual"] extra_HEAD_tags to ensure this test runs on CI.
+    extra_HEAD_tags = [],
+    # Override the default [] extra_mainnet_tags to skip this test on CI.
+    extra_mainnet_tags = ["manual"],
     flaky = False,
+    macro = rust_test_suite,
     proc_macro_deps = MACRO_DEPENDENCIES,
     deps = [":pocket-ic"] + DEPENDENCIES + TEST_DEPENDENCIES,
 )
 
-rust_test_suite(
+test_using_pocket_ic_server(
     name = "slow",
     size = "medium",
     srcs = ["tests/slow.rs"],
-    data = [
-        "//rs/pocket_ic_server:pocket-ic-server",
-    ],
-    env = {
-        "POCKET_IC_BIN": "$(rootpath //rs/pocket_ic_server:pocket-ic-server)",
-    },
+    # Override the default ["manual"] extra_HEAD_tags to ensure this test runs on CI.
+    extra_HEAD_tags = [],
+    # Override the default [] extra_mainnet_tags to skip this test on CI.
+    extra_mainnet_tags = ["manual"],
```

Comment:
Does "manual" mean that this test runs nightly? If so, it might fail once we add a new function to the library which does not work against the mainnet server.

Reply:
No "manual" means the test only runs when you call it explicitly using `bazel test`. So it won't run on any CI workflow.